### ISSUE TITLE
Check YouTube for HoA Migration for Curriculum Workshops

### ISSUE THREAD

**Issue by jgmac1106** [mozilla/learning.mozilla.org#2094]
HoA has finally left Google+ and moved to YouTube and you will not have access after Sept 12th. This does not affect the August call but for embedding in September you need to have AdSense active on the account.

Here is a quick tutorial video: https://www.youtube.com/watch?v=l6gneWIigRc

**Comment by hannahkane** [comment]
thanks, @jgmac1106.

@KGorr @chadsansing @amirad  - who should this be assigned to?

**Comment by KGorr** [comment]
That'd be me @hannahkane. Thanks, @jgmac1106!  

@ldecoursy - there are some legal-ese terms of service/monetization stuff that needs to be agreed to before we can move forward with YouTube Live. Who do we need to run that by before agreeing?

@hannahkane - assuming there are no snags, I just need to get you new embed codes for both the curriculum workshop and community call for the September pages.

**Comment by KGorr** [comment]
Essentially, to allow embedding with YouTube live (using our landing page), you have to enable monetization, which requires you to agree to be a YouTube Partner: https://support.google.com/youtube/answer/72851?hl=en @ldecoursy

**Comment by ldecoursy** [comment]
This probably needs legal review so please share with Abigail. 

Thanks for handling and @jgmac1106 thanks for the heads up!!

**Comment by KGorr** [comment]
Thanks, @ldecoursy. I'll keep everyone posted on the progress.

**Comment by KGorr** [comment]
Keeping everyone in the loop on progress here ... @HPaulJohnson just linked MoFo's AdSense account with our Mozilla Learning YouTube account so that we can enable live embedding. It's currently under a "under review" status. Once it gets approved by Google, we'll need to make sure ads stay turned off on our videos. If we _can't_ do that, we'll have more hoops to jump through. If we can, we'll be all set for broadcasting and I'll send along new embed codes for our Sept events.

**Comment by KGorr** [comment]
UPDATE: We've unfortunately exhausted all of our options to make YouTube Live work with our landing pages on learning.mozilla.org and have been unsuccessful.

**For September,** we will be using YouTube Live for both MCW & MCC, which will be broadcast ONLY on/from our YouTube channel. Viewers will have to join us on etherpad on another tab in their browser.

@hannahkane - can we remove the live embed section of both pages and replace with a note that says we've moved to YouTube (and link to our channel: https://www.youtube.com/channel/UC64W_SqCxhcTUwVpuSQWjEg) MCW is most urgent as it is happening next Tuesday. I will file a separate ticket for this too.

**For October,** I will be filing a ticket for us to get the ball rolling with AirMo, which means @hannahkane and team will have to work with that team to figure out the live embed options. If all goes really well with YouTube Live, we MAY cancel this ticket and continue to use YouTube Live. We want to have more use experience & info from a month of events before officially making that call.

If anyone has questions, please let me know. @ldecoursy @chadsansing @amirad Once all other tickets are opened, I'll close this one.

**Comment by hannahkane** [comment]
cc @simonwex for ambient awareness about switch to AirMo.

**Comment by hannahkane** [comment]
@kgorr - I think we can close this now, right? Please re-open if there's more to be done.
